mLRS/Common/setup.h

```diff
@@ -1,1046 +1,1086 @@
-    ee_status = ee_init();
-    if (ee_status == EE_STATUS_OK) { ee_status = setup_retrieve_from_EEPROM(); }
-    if (ee_status != EE_STATUS_OK) { // try it a 2nd time
-        setup_clear();
-        ee_status = ee_init();
-        if (ee_status == EE_STATUS_OK) { ee_status = setup_retrieve_from_EEPROM(); }
-    }
-
-    if (ee_status != EE_STATUS_OK) setup_clear(); // force default
-
-    if ((strncmp(Setup.MarkerStr, SETUP_MARKER_NEW_STR, 16) != 0) &&
-        (strncmp(Setup.MarkerStr, SETUP_MARKER_OLD_STR, 16) != 0)) {
-        setup_clear(); // force default
-    }
-
-#ifdef SETUP_FORCE_COMMON_CONF
-    setup_clear(); // force default
-#endif
-
-    doEEPROMwrite = false;
-    if (Setup.Layout != SETUPLAYOUT) {
-        if (Setup.Layout < SETUPLAYOUT) {
-            // was 335 or lower, there would be lots to do but didn't do layout version-ing properly so far
-
-            // Note: In v1.3.05 Mode changed, so we would have to change to MODE_19HZ_7X if it's a Sx127x.
-            // Luckily, the sanitizer will do that for us so we do not need to bother here.
-
-        } else {
-            // ups, > 10304, should not happen
-            for (uint8_t id = 0; id < SETUP_CONFIG_NUM; id++) setup_default(id);
-            Setup._ConfigId = 0;
-        }
-        Setup.Layout = SETUPLAYOUT;
-        doEEPROMwrite = true;
-    }
-    if (Setup.Version != VERSION) { // do after Layout, ensures that these flags are correct irrespective of Layout handling
-        Setup.Version = VERSION;
-        doEEPROMwrite = true;
-    }
-    if ((strncmp(Setup.MarkerStr, SETUP_MARKER_NEW_STR, 16) != 0)) {
-        strbufstrcpy((char*)Setup.MarkerStr, SETUP_MARKER_NEW_STR, 16);
-        strbufstrcpy((char*)Setup.MarkerEnd, SETUP_MARKEREND_STR, 8);
-        doEEPROMwrite = true;
-    }
-    if (doEEPROMwrite) {
-        setup_store_to_EEPROM();
-    }
-
-#ifdef DEVICE_IS_TRANSMITTER
-    if (Setup._ConfigId >= SETUP_CONFIG_NUM) Setup._ConfigId = 0;
-
-    // ensure that all config id's have the same channel_source
-    // otherwise it can be very confusing concerning how to communicate with the module
-    for (uint8_t id = 0; id < SETUP_CONFIG_NUM; id++) {
-        Setup.Tx[id].ChannelsSource = Setup.Tx[Setup._ConfigId].ChannelsSource;
-    }
-#else // DEVICE_IS_RECEIVER
-    Setup._ConfigId = 0;
-#endif
-
-    Config.ConfigId = Setup._ConfigId;
-
-    setup_configure_metadata();
-
-    setup_sanitize_config(Config.ConfigId);
-
-    setup_configure_config(Config.ConfigId);
-}
-
-
-#endif // SETUP_H
-
-
-/*
-setup_configure_metadata();
-  -> allowed_mask
-  -> optst
-  -> SetupMetaData.rx_xxxx
-
-setup_sanitize_config(Config.ConfigId):
-  SANITIZE():   ensure param value is in possible range
-  TST_NOTALLOWED(): ensure param value is one of the allowed ones
-
-  adjust allowed_mask where it depends on setting
-
-setup_configure_config(Config.ConfigId):
-
-
-
-*/
+//*******************************************************
+// Copyright (c) MLRS project

```

Comment:
not sure what has happened here, but something went terribly wrong with setup.h., if marks all of the lines as being changed ...

pl repair that :)